src/java.base/share/classes/java/net/Socket.java

```diff
@@ -39,6 +41,7 @@
 import java.util.Set;
 import java.util.Collections;
 
+
```

Comment:
Minor nit, I assume this additional blank line is left over from a previous iteration.